go/libkb/rpc_exim.go

```diff
@@ -1393,7 +1397,7 @@ func (e ChatInternalError) ToStatus() keybase1.Status {
 func (e ChatConvExistsError) ToStatus() keybase1.Status {
 	kv := keybase1.StringKVPair{
 		Key:   "ConvID",
-		Value: e.ConvID.String(),
+		Value: hex.EncodeToString(e.ConvID),
```

Comment:
Hmm, I'm not sure I agree. `String` is used all over the place on `ConversationID`, I think having it be different in one place is not a great play. I think the bigger problem here was the cast on the other side (which I think I put in).